### DISCUSSION TITLE
yq command not found : Bitbucket

### DISCUSSION THREAD

**Discussion by Ronak-Toshniwal** [mikefarah/yq#1499] [Q&A]
HI

> image: atlassian/default-image:3
> pipelines:
>  #pull-requests:
>  branches:
>   'Dev': 
>     - step:
>         script:
>           #Assign variables
>           - export CLIURL=https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz
>           - export SFDX_AUTOUPDATE_DISABLE=false
>           - export SFDX_USE_GENERIC_UNIX_KEYCHAIN=true
>           - export SFDX_DOMAIN_RETRY=300
>           - export SFDX_DISABLE_APP_HUB=true
>           - export SFDX_LOG_LEVEL=DEBUG
>           - export DEPLOYDIR=src
>           - export TESTLEVEL=RunLocalTests
>           #Create sfdx directory
>           - mkdir ~/sfdx
>           #Install Salesforce CLI
>           - npm install sfdx-cli
>           - export PATH=~/sfdx/bin:$PATH
>           - sfdx --version
>           - sfdx plugins --core
> 
>           
>           # Install utilities
>           
>           #- pip install yq
>           - npm install yq
>           - yq --version
>      

giving this error 

<img width="781" alt="image" src="https://user-images.githubusercontent.com/73140231/210359608-f30a6d36-56c4-4927-95c0-70fb1d4f4ac7.png">

**Comment by mikefarah** [comment]
yq is not a javascript library, so it's not installed via npm. See the readme page on how you can get yq
